subsys/bluetooth/mesh/friend.c

```diff
@@ -872,23 +879,39 @@ int bt_mesh_friend_req(struct bt_mesh_net_rx *rx, struct net_buf_simple *buf)
 	return 0;
 }
 
+static bool is_seg(struct bt_mesh_friend_seg *seg, u16_t src, u16_t seq_zero)
+{
+	struct net_buf *buf = (void *)sys_slist_peek_head(&seg->queue);
+
+	if (!buf) {
+		return false;
+	}
+	struct net_buf_simple_state state;
```

Comment:
Variable declaration to the beginning of the scope (the function in this case). Also, always empty line after `{` unless the next line is also a `{`.